gcp-ts-serverless-raw/README.md

```diff
@@ -1,6 +1,6 @@
 [![Deploy](https://get.pulumi.com/new/button.svg)](https://app.pulumi.com/new?template=https://github.com/pulumi/examples/blob/master/gcp-ts-serverless-raw/README.md)
 
-# Google Cloud Functions in Python and Go Deployed with TypeScript
+# Google Cloud Functions in Python, Go, and Typescript Deployed with TypeScript
```

Comment:
```suggestion
# Google Cloud Functions in Python, Go, and TypeScript Deployed with TypeScript
```

Reply:
Would you mind adding TypeScript to the `description` field here as well? https://github.com/pulumi/examples/blob/0dd0c484ee335693950fb701be0255cc11d915db/gcp-ts-serverless-raw/Pulumi.yaml#L3